src/Core/src/Handlers/Border/BorderHandler.Android.cs

```diff
@@ -46,9 +46,9 @@ void UpdateContent()
 				NativeView.AddView(view.ToPlatform(MauiContext));
 		}
 
-		public static void MapContent(BorderHandler handler, IBorderView border)
+		public static void MapContent(IBorderHandler handler, IBorderView border)
 		{
-			handler.UpdateContent();
+			(handler as BorderHandler)?.UpdateContent();
```

Comment:
For this one I'd say that `UpdateContent` doesn't really need to be a member method.  It's not using any state on the Handler so we could just make UpdateContent static here and pass in `NativeView` and `VirtualView`

`UpdateContent(handler.NativeView, handler.VirtualView)